src/neuroconv/datainterfaces/behavior/video/externalvideointerface.py

```diff
@@ -87,13 +91,26 @@ def get_metadata_schema(self):
             "required": [self.video_name],
             "additionalProperties": True,
         }
+        metadata_schema["properties"]["Behavior"]["required"].append("ExternalVideoDevices")
+        metadata_schema["properties"]["Behavior"]["properties"]["ExternalVideoDevices"] = {
+            "type": "object",
+            "properties": {self._default_device_name: device_metadata_schema},
+            "additionalProperties": True,
+        }
         return metadata_schema
 
     def get_metadata(self):
         metadata = super().get_metadata()
         video_metadata = {
             "Behavior": {
-                "ExternalVideos": {self.video_name: dict(description="Video recorded by camera.", unit="Frames")}
+                "ExternalVideos": {
+                    self.video_name: dict(
+                        description="Video recorded by camera.", unit="Frames", device=self._default_device_name
+                    )
+                },
+                "ExternalVideoDevices": {
```

Comment:
Do you disagree with having the device metadata nested inside the corresponding video?

I was thinking on something like this:

```python
video_metadata = {
    "Behavior": {
        "ExternalVideos": {
            self.video_name: dict(
                description="Video recorded by camera.",
                unit="Frames",
                device={
                    "name": self._default_device_name,
                    "description": "Video camera used for recording."
                },
            )
        }
    }
}
```

Reply:
Actually, that looks very nice, I wasn't understanding what you were asking for earlier. Done!